template-ui/src/components/CardBody.vue

```diff
@@ -1,19 +1,21 @@
 <template>
-  <div>
-    <p class="text-uppercase text-info mb-1">
+  <div class="card-content">
+    <p class="text-uppercase text-info mb-1 card-type text-truncate">
       <span v-if="typeTag">{{ typeTag }}</span>
       <span v-if="typeTag && gradeOrLevelTag"> • </span>
       <span v-if="gradeOrLevelTag">{{ gradeOrLevelTag }}</span>
     </p>
-    <h5 class="mb-1">
+    <h5 class="mb-1 card-title">
```

Comment:
Same here, card-title isn't used.
```suggestion
    <h5 class="mb-1">
```